gator_versioned_docs/version-0.2.0/guides/smart-accounts/send-gasless-transaction.md

```diff
@@ -13,10 +13,10 @@ You can use any paymaster service provider, such as [Pimlico](https://docs.pimli
 
 ## Prerequisites
 
-- [Install and set up the Smart Accounts Kit.](../../get-started/install.md)
-- [Create a MetaMask smart account.](create-smart-account.md)
+- [Install and set up the Smart Accounts Kit](../../get-started/install.md)
+- [Create a MetaMask smart account](create-smart-account.md)
 
-## Send a gasless transaction
+## How to send a gasless transaction
```

Comment:
Same as above

Reply:
Having same headings didn't affect the SEO, so IMO it should be fine. The first is title, the second is TOC heading. "with a paymaster" sounds like there are multiple ways.